src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_core.clj

```diff
@@ -40,11 +40,11 @@
         changed-path (when-not (= :unknown kind)
                        (.context event))
         full-path (when-not (= :unknown kind)
-                    (-> watched-path (.resolve changed-path) fs/file))]
+                    (-> watched-path (.resolve changed-path) (.toFile)))]
     {:type kind
      :count (.count event)
-     :watched-path (fs/file watched-path)
-     :changed-path (fs/file changed-path)
+     :watched-path (.toFile watched-path)
+     :changed-path (when changed-path (.toFile changed-path))
```

Comment:
> We cannot call `me.raynes.fs/file` on the relative path provided by
`WatchEvent#context()` without it being resolved to an absolute path
relative to the CWD, which is incorrect behavior.

This sentence doesn't seem to make sense to me.  By definition, absolute paths aren't relative to any directory, that's what makes them "absolute".

Is something actually broken or is this just about a desire to have these paths be relative instead of absolute?  If it's the former, shouldn't this break (all of?) the tests?  If the latter, I think I'm in support of that, but if it's that important it might be worth documenting somewhere, such as the docstring on the `Event` schema.